pcsx2/GS/Renderers/HW/GSRendererHW.cpp

```diff
@@ -5374,22 +5365,25 @@ __ri void GSRendererHW::DrawPrims(GSTextureCache::Target* rt, GSTextureCache::Ta
 			if (rt_new_alpha_max == rt_new_alpha_min)
 				rt->m_alpha_range = false;
 		}
+
+		if (DATE)
+		{
+			if (m_cached_ctx.TEST.DATM)
+			{
+				blend_alpha_min = std::max(blend_alpha_min, 128);
+				blend_alpha_max = std::max(blend_alpha_max, 128);
+			}
+			else
+			{
+				blend_alpha_min = std::min(blend_alpha_min, 127);
+				blend_alpha_max = std::min(blend_alpha_max, 127);
+			}
```

Comment:
I still don't know how this is "cleaner" all you've done is added an extra comparison it has to process :P

Reply:
I mainly wanted to move it out of where it was, didn't fit.

Reply:
How did it not fit? The code it was with was for optimising DATE and that code was optimising the blend with DATE